searchwidget/src/main/res/values/colors.xml

```diff
@@ -52,4 +52,6 @@
     <color name="blue_light">#01adff</color>
     <color name="blue_dark">#3369e6</color>
 
+    <color name="last_result_color">@android:color/black</color>
```

Comment:
keep the color name as it is in the `name` field value. See the above lines for reference.

Reply:
Got it, changed the color name to "black".